### ISSUE TITLE
how to dislike a repo 🗿🗿🗿🥶🥶🥶

### ISSUE THREAD

**Issue by SpinThwomp** [moyaiu/OldCord#28]

**Comment by DaMoyai** [comment]
idk ask microsoft

**Comment by toydotgame** [comment]
at least this repo is better than YOU'R MOM!!!!!!!!!!

**Comment by DaMoyai** [comment]
> at least this repo is better than YOU'R MOM!!!!!!!!!!

haha 😂😂🤣 u go him!!1

**Comment by SpinThwomp** [comment]
>at least this repo is better than YOU'R MOM!!!!!!!!!!

*Yo'ruo'er get rekt 🥴

**Comment by DaMoyai** [comment]
> > at least this repo is better than YOU'R MOM!!!!!!!!!!
> 
> *Yo'ruo'er get rekt woozy_face

don't care + didn't ask + cry about it + stay mad + get real + L + mald seethe cope harder + hoes mad + basic + skill issue + ratio + you fell off + the audacity + triggered + soyjak + any askers + redpilled + get a life + ok and? + cringe + touch grass + donowalled + not based + your're a (insert stereotype) + you're* + grammar issue + go outside + get good + reported + ad hominem + GG! + ask deez + ez clap + straight cash + ratio again + final ratio + stay mad + stay pressed + pedophile + cancelled + done for + mad free + freer than air + rip bozo + slight_smile + cringe again + mad cuz bad + lol + irrelevant + cope + jealous + go ahead whine about it + your problem + don't care even more + sex offender + sex defender + not okay + glhf + problematic + you're white + youre British + no u + deez nuts + radio + I'm a minor + caught in 4k + in 1947 the worlds first general purpose computer, the 30 ton ENIAC was created + your moms + your dad + your grandma + the hood watches markiplier now + grow up + L + L (part two) + incel + virgin + ligma + taco bell tortilla crunch + ur dad fell off + the government experimented on innocent black men in a lab + think outside the bun + my dad could beat ur dad up + ur aimhacking + try hard + (insert insult) + Karen + ok boomer + sksksk + hyperpop isnt real music + ur beta + im sigma + ur submissive + L (part 3) + ur sus + me when the imposter is sus + this is a cry for help and im depressed + quote tweet + you bought monkey nft + you're weird champ + my dad owns (insert game development company) + nice Mac + pack watch + go find a father figure + this is a cry for help + NFT buyers when they discover save image + the earth is round + cheetos breath + dream isnt a real artist + [r/woooosh](https://reddit.com/r/woooosh) + pronouns in bio + anime pfp + so why do good girls like bad guys + employee of the month + invest in bitcoin + stop pmsing + not funny didnt laugh + neutral_face + ez clap + vape crack User + Chile Predator on Omegle + Touch grass instead of writing these textes + get a Life + one more Word and you are blocked

**Comment by SpinThwomp** [comment]
🤓

**Comment by DaMoyai** [comment]
No, actually, that is not me. I am not an emoji- nor do I resemble one- I am a human. Also who are you to tell me what I look like when, in fact, you have never seen me nor likely never will? All that aside, it's clear that you are trying to insult me by calling me a "nerd" (as depicted by the emoji you have sent). I rather take your "insult" as a compliment, as calling me a "nerd", a word often used to mock smarter people, implies that i am more intelligent and have a larger vocabulary than you. Next time you disagree, I recommend you try to come up with an actual argument, or at least prepare yourself one for when necessary. Additionally, before you proceed with sending me a clown emoji (admitting defeat), just understand that you would be further praising me, as a clown is someone who is entertaining and usually considered humorous by the spectators. Furthermore, you would also be implying that I, employed by a business to perform as a clown, have a profession. So, not only are you implying that I am smart, you are also implying that I am funny and have a job, which, judging by; your lack of proper arguments, your usage for childish and overused internet humor, the amount of spare time you have to brainwash yourself 12 hours per day with social media culture; you are neither intelligent, comedic, or employed. Enjoy watching yourself to continue on through life as such, as I believe with full confidence that you lack the decency to better your ruined self for the real world. Carry on and have a pleasant day.

**Comment by SpinThwomp** [comment]
thats a nice argument, unfortunately: your mother

**Comment by DaMoyai** [comment]
That's a nice argument you have, but unfortunately, I had sexual intercourse with your mother, therefore rendering any point you make null.

**Comment by SpinThwomp** [comment]
that would have been a great argument, unfortunately for you, I have two dads 😐

**Comment by DaMoyai** [comment]
> that would have been a great argument, unfortunately for you, I have two dads neutral_face

unfortunately for you, you are adopted, proofing your biological parents died or disowned you, showing that you are a failure.

**Comment by SpinThwomp** [comment]
*proving

learn to spell buddy 🤡

**Comment by DaMoyai** [comment]
"learn to spell buddy 🤡"🤓

**Comment by SpinThwomp** [comment]
>"learn to spell buddy 🤡"🤓

- 🤓2022

**Comment by toydotgame** [comment]
twitter

**Comment by DaMoyai** [comment]
> > "learn to spell buddy clown_face"nerd_face
> 
> –nerd_face2022

![twitter user](https://user-images.githubusercontent.com/51537072/193447434-c4860fa9-5230-46cc-9e15-644dcb1805fe.png)
bro this u?

**Comment by SpinThwomp** [comment]
POV: DaMoyai looks into a mirror

**Comment by toydotgame** [comment]
actually image is from average oldcord hater wikipedia article [https://en.wikipedia.org/wiki/SpinThwomp](https://uncyclopedia.com/wiki/Your_mom)

**Comment by DaMoyai** [comment]
![sdfxvfsfvxcv](https://user-images.githubusercontent.com/51537072/194089746-56dc244f-f8ee-4222-b931-b30f89590cb1.png)
found spinthwomps alt

**Comment by toydotgame** [comment]
that's not a real person. like birds, comp sci students don't exist. they're a ploy made up by the government 🤪 #truth

**Comment by SpinThwomp** [comment]
> ![sdfxvfsfvxcv](https://user-images.githubusercontent.com/51537072/194089746-56dc244f-f8ee-4222-b931-b30f89590cb1.png)
> found spinthwomps alt

oh the one I use to get with your mother?

**Comment by toydotgame** [comment]
oh is that like the one your mother works at? (laugh at my incredibly cultured reference)

anyway + ratio + i did ur mom

**Comment by DaMoyai** [comment]
> > ![sdfxvfsfvxcv](https://user-images.githubusercontent.com/51537072/194089746-56dc244f-f8ee-4222-b931-b30f89590cb1.png)
> > found spinthwomps alt
> 
> oh the one I use to get with your mother?

oh so you're necrophilic

**Comment by SpinThwomp** [comment]
thats why yo mama dead 😂

**Comment by toydotgame** [comment]
yo momma so dandelion she blew away into a million bits and gone forever.

**Comment by foursplaysroblox** [comment]
![QuandaleDingleGIF](https://user-images.githubusercontent.com/82078139/210171950-8489d740-a417-4c78-82ac-b10c39708031.gif)

**Comment by foursplaysroblox** [comment]
What's up guys! It's Quandale Dingle here! (RUUEHEHEHEHEHEEHE) I have been arrested for multiple crimes (AHHHHHHHHHHHHH) including: Battery on a police officer (WHAT), Grand theft, Declaring war on Italy, and public indecency (RUHEHEHEEHEHEHEHEHEHEHE X2 speed). I will be escaping prison on, MARCH 28TH! After that.... I WILL TAKE OVER THE WORLD

Greetings! Quandale Dingle here. My cousin Henry Dinglenut got arrested for putting TNT in a daycare center. (WHAT THE FUCK??) I put a camera in Joe Biden's bathroom and watched him take a poop. (WHAT????) My Asian brother, Quanliling Dingle put illegal substances in my ramen and I died.

Hey, Quandale Dingle here. I just escaped prison and staying at Juandale Pringle's house. As I was running away from cops, I fell and scraped some of my foreskin off. A guy named Garfield Jenson bit me over in the shower while I was in prison. My baby momma Shiniqua Inderson told me to pay child support so I gave my baby to a creepy old guy,

Hey fellas it's Quandele Dingle here I put perks on Vladimir Putin's drink And he went to bed for a really long time I trapped my autistic son's hand in an air fryer (herrrrr) I dumped boiling water on a prison guard's head My goofy ah friend, Jamarious Quandele Dingle tried to eat my but during Ramadhan

What's up guys! It's Quandale Dingle here! (RUUEHEHEHEHEHEEHE) I have been arrested for multiple crimes (AHHHHHHHHHHHHH) including: Battery on a police officer (WHAT), Grand theft, Declaring war on Italy, and public indecency (RUHEHEHEEHEHEHEHEHEHEHE X2 speed). I will be escaping prison on, MARCH 28TH! After that.... I WILL TAKE OVER THE WORLD

Greetings! Quandale Dingle here. My cousin Henry Dinglenut got arrested for putting TNT in a daycare center. (WHAT THE FUCK??) I put a camera in Joe Biden's bathroom and watched him take a poop. (WHAT????) My Asian brother, Quanliling Dingle put illegal substances in my ramen and I died.

Hey, Quandale Dingle here. I just escaped prison and staying at Juandale Pringle's house. As I was running away from cops, I fell and scraped some of my foreskin off. A guy named Garfield Jenson bit me over in the shower while I was in prison. My baby momma Shiniqua Inderson told me to pay child support so I gave my baby to a creepy old guy,

Hey fellas it's Quandele Dingle here I put perks on Vladimir Putin's drink And he went to bed for a really long time I trapped my autistic son's hand in an air fryer (herrrrr) I dumped boiling water on a prison guard's head My goofy ah friend, Jamarious Quandele Dingle tried to eat my but during Ramadhan

What's up guys! It's Quandale Dingle here! (RUUEHEHEHEHEHEEHE) I have been arrested for multiple crimes (AHHHHHHHHHHHHH) including: Battery on a police officer (WHAT), Grand theft, Declaring war on Italy, and public indecency (RUHEHEHEEHEHEHEHEHEHEHE X2 speed). I will be escaping prison on, MARCH 28TH! After that.... I WILL TAKE OVER THE WORLD

Greetings! Quandale Dingle here. My cousin Henry Dinglenut got arrested for putting TNT in a daycare center. (WHAT THE FUCK??) I put a camera in Joe Biden's bathroom and watched him take a poop. (WHAT????) My Asian brother, Quanliling Dingle put illegal substances in my ramen and I died.

Hey, Quandale Dingle here. I just escaped prison and staying at Juandale Pringle's house. As I was running away from cops, I fell and scraped some of my foreskin off. A guy named Garfield Jenson bit me over in the shower while I was in prison. My baby momma Shiniqua Inderson told me to pay child support so I gave my baby to a creepy old guy,

Hey fellas it's Quandele Dingle here I put perks on Vladimir Putin's drink And he went to bed for a really long time I trapped my autistic son's hand in an air fryer (herrrrr) I dumped boiling water on a prison guard's head My goofy ah friend, Jamarious Quandele Dingle tried to eat my but during Ramadhan

What's up guys! It's Quandale Dingle here! (RUUEHEHEHEHEHEEHE) I have been arrested for multiple crimes (AHHHHHHHHHHHHH) including: Battery on a police officer (WHAT), Grand theft, Declaring war on Italy, and public indecency (RUHEHEHEEHEHEHEHEHEHEHE X2 speed). I will be escaping prison on, MARCH 28TH! After that.... I WILL TAKE OVER THE WORLD

Greetings! Quandale Dingle here. My cousin Henry Dinglenut got arrested for putting TNT in a daycare center. (WHAT THE FUCK??) I put a camera in Joe Biden's bathroom and watched him take a poop. (WHAT????) My Asian brother, Quanliling Dingle put illegal substances in my ramen and I died.

Hey, Quandale Dingle here. I just escaped prison and staying at Juandale Pringle's house. As I was running away from cops, I fell and scraped some of my foreskin off. A guy named Garfield Jenson bit me over in the shower while I was in prison. My baby momma Shiniqua Inderson told me to pay child support so I gave my baby to a creepy old guy,

Hey fellas it's Quandele Dingle here I put perks on Vladimir Putin's drink And he went to bed for a really long time I trapped my autistic son's hand in an air fryer (herrrrr) I dumped boiling water on a prison guard's head My goofy ah friend, Jamarious Quandele Dingle tried to eat my but during Ramadhan
